### ISSUE TITLE
Remove the 'Manually Save Representation' step when joining meshes

### ISSUE THREAD

**Issue by theoryshaw** [IfcOpenShell/IfcOpenShell#4349]
Would it be possible to remove the 'Manually Save Representation' step, when joining meshes?

https://github.com/IfcOpenShell/IfcOpenShell/assets/507113/429a80ce-695a-4684-945e-606af3b5bcdc

**Comment by Gorgious56** [comment]
There is an IFC join override that will preserve both meshes, try F3 and type "Ifc Join". 
I think it's also by default what happens if you use the shortcut CTRL + J.

![image](https://github.com/IfcOpenShell/IfcOpenShell/assets/25156105/5b24298c-a829-4c6e-a9ba-e94b9b5a8358)

**Comment by theoryshaw** [comment]
Thanks @Gorgious56, good to know!

Think it worth changing the intent of this ticket to improve the UI in the regard?  I think the current joining work-around is a little confusing to a lot of Blender users coming to BB.

**Comment by Gorgious56** [comment]
Yup the problem I think is that the right click context menu is hardcoded, we can only prepend or append elements, not modify existing ones.

**Comment by Gorgious56** [comment]
Adding this so it tracked.

Users need to enable "Developer Extras" in the preferences to be able to see the shortcuts. The reason is because only operators that have been added to a UI element (panel, menu, header) appear in the search queries by default. I think it's dumb but the devs decided so a few version ago. Maybe we can add it to a hidden menu somewhere so it pops up in users search queries. Would be a first step for discoverability

**Comment by BIMvoice** [comment]
@Gorgious56 your diagnosis still holds exactly: "IFC Join" (bim.override_object_join) exists and Ctrl+J is bound to it, but the right-click context menu never included it, so users fall through to Blender's native non-IFC-aware join. Proposed in #8619: added it to the existing appended object menu, right alongside the other IFC-aware override operators that are already there, so it's discoverable without needing Developer Extras + F3 search.

Generated with the assistance of an AI coding tool.
